python/moose/moose.py

```diff
@@ -58,16 +60,6 @@
     mergechemImport_ = False
     mergechemError_ = '%s' % e
 
-
-# Import function from C++ module into moose namespace.
-from moose._moose import *
-
```

Comment:
This was no longer required. See comment on previous file where I replace `moose` with `_moose`.